### DISCUSSION TITLE
Loading 3D models using NiceGUI

### DISCUSSION THREAD

**Discussion by v479038280** [zauberzeug/nicegui#1568] [Ideas / Feature Requests]
If we can load 3D models using nicegui, nicegui supports three.js ,and three.js supports [Loading 3D models](https://threejs.org/docs/manual/en/introduction/Loading-3D-models.html)

**Comment by rodja** [comment]
See https://github.com/zauberzeug/nicegui/blob/main/examples/3d_scene/main.py
